osmnames/import_osm/import_osm.py

```diff
@@ -0,0 +1,92 @@
+import glob
+import os
+from subprocess import check_call
+from osmnames.helpers.database import psql_exec, exec_sql_from_file
+from osmnames import settings
+
+
+def run():
```

Comment:
`run` seems a bit generic, what does it do? I would prefer something like `import` (or anything more specific than `run` ;-) )